### PR TITLE
Update xml-schema-definition-tool-xsd-exe.md

### PR DESCRIPTION
**Pull request by ptorr-msft**
Clarified the /f option

## Summary

Makes it clearer that fields are always generated, and /f just omits the properties.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ xsd {file.dll | file.exe} [-outputdir:directory] [/type:typename [...]][/paramet
...
 |**/enableDataBinding**|Implements the <xref:System.ComponentModel.INotifyPropertyChanged> interface on all generated types to enable data binding. The short form is `/edb`.|
 |**/enableLinqDataSet**|(Short form: `/eld`.) Specifies that the generated DataSet can be queried against using LINQ to DataSet. This option is used when the /dataset option is also specified. For more information, see [LINQ to DataSet Overview](../../framework/data/adonet/linq-to-dataset-overview.md) and [Querying Typed DataSets](../../framework/data/adonet/querying-typed-datasets.md). For general information about using LINQ, see [Language-Integrated Query (LINQ) - C#](../../csharp/programming-guide/concepts/linq/index.md) or [Language-Integrated Query (LINQ) - Visual Basic](../../visual-basic/programming-guide/concepts/linq/index.md).|
-|**/f\[ields\]**|Generates fields instead of properties. By default, properties are generated.|
+|**/f\[ields\]**|Generates fields only. By default, get / set properties are generated in addition to the backing fields.|
```
Comment (riverar) [inline docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md:62]: `get` and `set` are methods, not properties. Tweaked wording and added a backlink.

```suggestion
|**/f\[ields\]**|Generates fields only. By default, [properties with backing fields](https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/properties#properties-with-backing-fields).|
```

File: docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ xsd {file.dll | file.exe} [-outputdir:directory] [/type:typename [...]][/paramet
...
 |**/enableDataBinding**|Implements the <xref:System.ComponentModel.INotifyPropertyChanged> interface on all generated types to enable data binding. The short form is `/edb`.|
 |**/enableLinqDataSet**|(Short form: `/eld`.) Specifies that the generated DataSet can be queried against using LINQ to DataSet. This option is used when the /dataset option is also specified. For more information, see [LINQ to DataSet Overview](../../framework/data/adonet/linq-to-dataset-overview.md) and [Querying Typed DataSets](../../framework/data/adonet/querying-typed-datasets.md). For general information about using LINQ, see [Language-Integrated Query (LINQ) - C#](../../csharp/programming-guide/concepts/linq/index.md) or [Language-Integrated Query (LINQ) - Visual Basic](../../visual-basic/programming-guide/concepts/linq/index.md).|
-|**/f\[ields\]**|Generates fields instead of properties. By default, properties are generated.|
+|**/f\[ields\]**|Generates fields only. By default, [properties with backing fields](https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/properties#properties-with-backing-fields) are generated.|
```
Comment (tdykstra) [inline docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md:62]:
```suggestion
|**/f\[ields\]**|Generates fields only. By default, [properties with backing fields](../../csharp/programming-guide/classes-and-structs/properties.md#properties-with-backing-fields) are generated.|
```

**Review by tdykstra — APPROVED** [review]
LGTM, just need to change the link format.

File: docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md (hunk 2 above)
Comment (ptorr-msft) [reply docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md:62]: @tdykstra I don't know what the standard is for relative vs. absolute URLs. But there is no ".md" extension on the live website. You get 404s.

Comment (tdykstra) [reply docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md:62]: The docs build will translate the .md file reference to a docs URL.

**Pull request merged**